mlflow/environment_variables.py

```diff
@@ -595,10 +595,23 @@ def get(self):
     "MLFLOW_TRACING_CLIENT_BUFFER_SIZE", int, 1000
 )
 
-# How long a trace can be buffered at the in-memory trace client before being abandoned.
+# How long a trace can be "in-progress". When this is set to a positive value and a trace is
+# not completed within this time, it will be automatically halted and exported to the specified
+# backend destination with status "ERROR".
```

Comment:
```suggestion
#: How long a trace can be "in-progress". When this is set to a positive value and a trace is
#: not completed within this time, it will be automatically halted and exported to the specified
#: backend destination with status "ERROR".
```

I think `:`s are missing. Let's check if these environment variables appear in the preview.

Reply:
Ah great catch!